commands/env.go

```diff
@@ -182,6 +186,10 @@ func shellCfgUnset(c CommandLine, api libmachine.API) (*ShellConfig, error) {
 		shellCfg.Prefix = "(setenv \""
 		shellCfg.Suffix = ")\n"
 		shellCfg.Delimiter = "\" nil"
+	case "tcsh":
+		shellCfg.Prefix = "setenv "
```

Comment:
Any reason you didn't go with `unsetenv` instead? Downside of the current approach is it leaves envs around just blank:
```
% eval `./docker-machine env -u`
% setenv | grep DOCK
DOCKER_HOST=
DOCKER_TLS_VERIFY=
DOCKER_CERT_PATH=
DOCKER_MACHINE_NAME=
```